install/kibana-install.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: T.H. (ELKozel)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y apt-transport-https
```

Comment:
not needed, what is needed is the "core" dependencies.

Reply:
Be a lovely tteck :grinning:, tell him briefly what the “core” dependencies are for you. Otherwise he will never understand. :pray:

Reply:
If you look at every LXC script (`*-install.sh`), it should be easy to understand what the "core" dependencies are. He had them once, except he had `wget` instead of `curl` and I noted what needed to be changed.